### PR TITLE
Support acceptance test against OpenStack Rocky release in OpenLab

### PR DESCRIPTION
**Pull request by h00130372**
Input comment "recheck stable/rocky" in pull request comments, that
will trigger acceptance tests against OpenStack Rocky release. OpenLab
will report test result in followint comments automatically.

Related-Bug: theopenlab/openlab#82

### CODE REVIEW

**Comment by jtopjian** [comment]
@h00130372 Thanks! let me know when this is ready to merge.

**Comment by kiwik** [comment]
recheck

**Comment by kiwik** [comment]
recheck stable/rocky

**Comment by h00130372** [comment]
recheck stable/queens

**Comment by h00130372** [comment]
@jtopjian There are three failed tests:
1.TestRolesFilterList  --- the reason maybe is that we create a role with domain id(I found the commit merged several days ago), then we list roles without domain id, keystone will return the roles which without domain, so the test expects that the role can be found, but actually not.
2.the compute/v2 tests TestQuotasetGet and TestQuotasetUpdateDelete, I looked into the acceptance test, we use "github.com/gophercloud/gophercloud/openstack/identity/v2/tenants" to get the tenant, the keystone doesn't support the v2/tenants for several release? so "Resource not found" error raised.

 The ci against master is ok, because all of these tests we skip on the master branch "clients.SkipRelease(t, "master")".

**Comment by jtopjian** [comment]
@h00130372 okay, then go ahead and mark those tests to be skipped for rocky and queens.

**Comment by h00130372** [comment]
@jtopjian you don't want to fix them, just to skip? if so please propose a commit to skip them for rocky and  
queens, then we can merge this later, thank you.

**Comment by jtopjian** [comment]
> you don't want to fix them, just to skip?

Correct. 

Keystone is doing some internal changes and the Roles test might work in the future. For the Compute Quotas tests, that will take a little more work to fix which I don't have time for right now. So we can just skip the tests for now.

> if so please propose a commit to skip them for rocky and queens, then we can merge this later, thank you.

Since we're only skipping the tests, it's probably better for you to make those changes in this Pull Request. That way we can confirm that all problems are resolved rather than opening one (or possibly more) PRs.

If you would really prefer me to make the changes, I can do that.

**Comment by h00130372** [comment]
recheck

**Comment by h00130372** [comment]
recheck stable/queens

**Comment by h00130372** [comment]
recheck stable/rocky

File: .zuul.yaml [hunk 1]
```diff
@@ -60,7 +69,7 @@
     nodeset: ubuntu-trusty
 
 - project:
-    name: gophercloud/gophercloud
+    name: theopenlab/gophercloud
```
Comment (kiwik) [inline .zuul.yaml]: should be `gophercloud/gophercloud`

**Comment by h00130372** [comment]
recheck stable/queens

**Comment by h00130372** [comment]
recheck stable/rocky

**Comment by h00130372** [comment]
recheck stable/rocky

**Comment by h00130372** [comment]
recheck stable/rocky

**Comment by h00130372** [comment]
recheck stable/queens

**Comment by h00130372** [comment]
recheck

**Comment by h00130372** [comment]
recheck stable/rocky

**Comment by h00130372** [comment]
@jtopjian I skipped some tests, and the job run success, would you please to merge this pr, thank you!

**Comment by h00130372** [comment]
@jtopjian And I have a question about two tests: TestLayer3RouterInterface and TestLayer3FloatingIPsExternalCreateDelete, why we skip them against OpenStack master? they can not success against OpenStack Rocky, and I can not find the reason:( So I skip them for Rocky too.

**Comment by h00130372** [comment]
@jtopjian could you merge this pr please, thank you !

**Comment by h00130372** [comment]
recheck stable/rocky

**Comment by h00130372** [comment]
recheck

**Comment by h00130372** [comment]
recheck stable/rocky

**Comment by jtopjian** [comment]
@h00130372 Thank you for working on this! I'm tied up with a few other things, but I definitely intend to look at the tests you've noted as well as get this merged as soon as I can.

**Comment by h00130372** [comment]
@jtopjian Thank you! The master and rocky jobs are success, so we don't need to skip the tests I mentioned above. Wait for merging:)

**Pull request merged**